cfgov/unprocessed/css/owning-a-home/module/breadcrumbs.less

```diff
@@ -0,0 +1,36 @@
+.breadcrumbs {
+    .grid_column( 1, 1 );
+
+    padding-top: unit( @grid_gutter-width / 14px, em );
+    font-size: unit( 14px / @base-font-size-px, em );
+    overflow-x: auto;
+    white-space: nowrap;
+
+    &:before {
+      color: @pacific;
+      content: @cf-icon-left;
+      font-family: 'CFPB Minicons';
+    }
+
+    .respond-to-max( @mobile-max, {
+        padding-top: unit( @grid_gutter-width / 2 / 14px, em );
+        padding-bottom: unit( @grid_gutter-width / 2 / 14px, em );
+    } );
+
+    &_link {
+        padding-left: 4px;
+
+        &:not( :last-child ):after {
+            padding-left: 4px;
+            color: @text;
+            content: ' /';
+        }
+    }
+}
+
+// Hide on print.
```

Comment:
File was removed in https://github.com/cfpb/cfgov-refresh/pull/3165/commits/adea54f8ecb4f983b08882aaf842c2e812e402bf.